src/App.tsx

```diff
@@ -1,26 +1,53 @@
+import { FC, useEffect, useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { setTodos } from './features/todos';
+import { selectCurrentTodo } from './features/currentTodo';
+
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App: FC = () => {
+  const dispatch = useAppDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const currentTodo = useAppSelector(selectCurrentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setIsLoading(true);
+
+    getTodos()
+      .then(data => {
+        dispatch(setTodos(data));
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, [dispatch]);
+
+  return (
+    <>
+      <div className="section">
+        <div className="container">
+          <div className="box">
+            <h1 className="title">Todos:</h1>
+
+            <div className="block">
+              <TodoFilter />
+            </div>
+
+            <div className="block">
+              {isLoading && <Loader />}
 
-          <div className="block">
-            <Loader />
-            <TodoList />
+              <TodoList />
```

Comment:
Verify that the TodoList component is correctly receiving and displaying the todos from the state. Ensure that the state is properly mapped to the props of TodoList.